wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ViewModel/ConversationListViewControllerViewModel.swift

```diff
@@ -199,7 +206,7 @@ extension ConversationListViewController.ViewModel {
         return true
     }
 
-    private func updateE2EICertifiedStatus() {
+    func updateE2EICertifiedStatus() {
```

Comment:
To support concurrency also for other calls I would rather provide:
```swift
    func updateE2EICertifiedStatus() {
        Task {
            updateE2EICertifiedStatus()
        }
    }

    @MainActor
    private func updateE2EICertifiedStatus() async {
        do {
            selfUserStatus.isE2EICertified = try await isSelfUserE2EICertifiedUseCase.invoke()
        } catch {
            WireLogger.e2ei.error("failed to get E2EI certification status: \(error)")
        }
    }
```

Reply:
This intentionally runs in the background, currently there's no need to await the use case.

Reply:
I have used it for my suggestion of notification token ;) okay now there is no case anymore 👍